### ISSUE TITLE
Grouping slice is not taking the next color applied to the series

### ISSUE THREAD

**Issue by SDVII** [amcharts/amcharts5#1124]
**Bug description**

In theory, the grouping slice should take the next color from the colors applied to a series, but it seems that it is currently reusing the first color instead. Here is [code pen](https://codepen.io/SDVII/pen/abPPeNP) reproducing the issue. Am I missing anything with the setup?

In the example I provided 8 colors equal to the number of slices, unless I need to have a set of colors equal to all ungrouped slices + 1?

I've currently utilized adapters to set the grouping slice's color, but it's really dirty:
```js
series.slices.template.adapters.add("fill", function(color, target) {
    return target.dataItem === grouper.getPrivate("groupDataItem") ? groupSliceColor : color;
 });
```

Is there an official way to avoid this? 

**Environment (if applicable)**

 - amCharts: 5.3.8
 - Browser: Edge v117.0.2045.60
 - React: v18.2.0

**Comment by martynasma** [comment]
Fixed in [5.4.11](https://github.com/amcharts/amcharts5/releases/tag/5.4.11).

## [5.4.11] - 2023-10-10

### Fixed
- `SliceGrouper` would use first color in series for a "group" slice if series was using a color set with custom list of colors.
- Fixed a problem where no series was displayed after data with all 0 values was set (since `5.4.10`).
- `forceInactive` was not working on `Container`.
- `drawingsInteractive(value: bool)` method in `StockChart` was not working properly.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.

**Comment by SDVII** [comment]
The fix works, thank you for your hard work!
